DiscordBot/Commands/EventCommands.cs

```diff
@@ -136,11 +141,32 @@ await context.Dependencies.GetDependency<IEventsSheetsService>()
                 }
                 catch (EventNotFoundException)
                 {
-                    await context.RespondAsync($"{context.Member.Mention} - operation stopped: event not found");
+                    await context.RespondAsync($"{context.Member.Mention} - operation stopped: event not found.");
                 }
             }
         }
 
+        public async Task ListEvents(CommandContext context, InteractivityModule interactivity)
+        {
+            var eventsSheetsService = context.Dependencies.GetDependency<IEventsSheetsService>();
+            var eventsList = await eventsSheetsService.ListEventsAsync();
+
+            var eventsListEmbed = new DiscordEmbedBuilder
+            {
+                Title = "Events"
+            };
+
+            foreach (var discordEvent in eventsList)
+            {
+                eventsListEmbed.AddField(
+                    $"{discordEvent.Key}) {discordEvent.Name}",
+                    $"{discordEvent.Time}"
+                );
+            }
+
+            await context.RespondAsync($"{context.Member.Mention} - here are the events.", embed: eventsListEmbed);
```

Comment:
Maybe something like "Here are all created events" might be a bit more descriptive? 🙂